ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -366,32 +368,42 @@ instance CardanoHardForkConstraints c => CanHardFork (CardanoEras c) where
           PCons translateLedgerStateByronToShelleyWrapper
         $ PCons translateLedgerStateShelleyToAllegraWrapper
         $ PCons translateLedgerStateAllegraToMaryWrapper
+        $ PCons translateLedgerStateMaryToAlonzoWrapper
         $ PNil
     , translateChainDepState =
           PCons translateChainDepStateByronToShelleyWrapper
         $ PCons translateChainDepStateAcrossShelley
         $ PCons translateChainDepStateAcrossShelley
+        $ PCons translateChainDepStateAcrossShelley
         $ PNil
     , translateLedgerView    =
           PCons translateLedgerViewByronToShelleyWrapper
         $ PCons translateLedgerViewAcrossShelley
         $ PCons translateLedgerViewAcrossShelley
+        $ PCons translateLedgerViewAcrossShelley
         $ PNil
     }
   hardForkChainSel =
         -- Byron <-> Shelley, ...
-        TCons (CompareBlockNo :* CompareBlockNo :* CompareBlockNo :* Nil)
```

Comment:
Can we change this to `hcPure @(Proxy ____) CompareBlockNo`? If the `___` is too obfuscatory, we maybe should leave it as-is.

Reply:
Maybe we could do similar for the Shelley eras too.